db/01_create_tweets_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE tweets(
```

Comment:
I would add some database setup info maybe here on in the README - to avoid people calling the database the wrong thing